scripts/mina-local-network/mina-local-network.sh

```diff
@@ -769,7 +869,6 @@ if [ "${SNARK_WORKERS_COUNT}" -gt 0 ]; then
 		Instance #0:
 		  pid ${SNARK_COORDINATOR_PID}
 		  status: ${MINA_EXE} client status -daemon-port ${SNARK_COORDINATOR_PORT}
-		  logs: cat ${NODES_FOLDER}/snark_coordinator/log.txt | ${LOGPROC_EXE}
```

Comment:
Actually I want to protest :D This print was quite useful so i can quickly view logs . I understand that you removed logproc as it was only used for this informational output. However, path to log or even to working dir for particular service would be useful

Reply:
This is stdout dumped to file. And I've removed the dumping, I'll add back the mina config root as a hint for where to find file log FWIW